qa/suites/rgw/verify/tasks/rgw_ragweed.yaml

```diff
@@ -0,0 +1,25 @@
+os_type: centos   # xenial valgrind buggy, see http://tracker.ceph.com/issues/18126
+tasks:
+- install:
+    flavor: notcmalloc
+    debuginfo: true
+- ceph:
+- rgw:
+    client.0:
+      valgrind: [--tool=memcheck]
+- ragweed:
+    client.0:
+      force-branch: ceph-master
+      rgw_server: client.0
+      stages: prepare
+- ragweed:
+    client.0:
+      force-branch: ceph-master
+      rgw_server: client.0
+      stages: check
```

Comment:
can this not just use a single `ragweed` task with `stages: check, prepare`?  (same with `multifs/tasks/rgw_ragweed.yaml`)

Reply:
@cbodley we can, but this lays out the template for multi-stage runs which are more interesting.